src/Tests/database.test.js

```diff
@@ -0,0 +1,6 @@
+try {
```

Comment:
This file is not being used too! make sure to remove it or it is being used properly.

Reply:
Am using it soon I just haven't moved to testing